admin-dev/themes/default/template/helpers/options/options.tpl

```diff
@@ -78,7 +78,7 @@
 						<div class="col-lg-12">
 							<p class="help-block">
 								<strong>{l s='Check / Uncheck all'}</strong><br />
-								{l s='You are editing this page for a specific shop or group. Click "Yes" to check all fields, "No" to uncheck all.'}<br />
+								{l s='You are editing this page for a specific shop or group. Click "%yes_label%" to check all fields, "%no_label%" to uncheck all.' d='Admin.Design.Help' sprintf=['%yes_label%' => {l s='Yes' d='Admin.Global', '%no_label%' => {l s='No' d='Admin.Global'}]}<br />
```

Comment:
Admin.Design.Help?

Reply:
It's the same wording as in `src/PrestaShopBundle/Resources/views/Admin/macros.html.twig` which uses `Admin.Design.Help` So I thought it was better to use the same domain in both files to avoid duplicate wording

Reply:
Alright, thanks!